pvlib/spectrum/mismatch.py

```diff
@@ -689,6 +690,106 @@ def spectral_factor_pvspec(airmass_absolute, clearsky_index,
     return mismatch
 
 
+def spectral_factor_jrc(airmass, clearsky_index, module_type=None,
+                        coefficients=None):
+    r"""
+    Estimate a technology-specific spectral mismatch modifier from absolute
+    airmass and clear sky index using the JRC model.
+
+    The JRC spectral mismatch model includes the effects of cloud cover on
+    the irradiance spectrum. Model coefficients are derived using measurements
+    of irradiance and module performance at the Joint Research Centre (JRC) in
+    Ispra, Italy (45°48'N, 8°37'E). Coefficients for two module types are
+    available via the ``module_type`` parameter. More details on the model can
+    be found in [1]_.
+
+    Parameters
+    ----------
+    airmass_absolute : numeric
+        absolute (pressure-adjusted) airmass. [unitless]
+
+    clearsky_index: numeric
+        clear sky index. [unitless]
+
+    module_type : str, optional
+        One of the following PV technology strings from [1]_:
+
+        * ``'cdte'``, - anonymous CdTe module.
+        * ``'multisi'`` - anonymous multicrystalline Si module.
+
+    coefficients : array-like, optional
+        user-defined coefficients, if not using one of the default coefficient
+        sets via the ``module_type`` parameter.
+
+    Returns
+    -------
+    mismatch: numeric
+        spectral mismatch factor (unitless) which is multiplied
+        with broadband irradiance reaching a module's cells to estimate
+        effective irradiance, i.e., the irradiance that is converted to
+        electrical current.
+
+    Notes
+    -----
+    The JRC model parameterises the spectral mismatch factor as a function
+    of absolute air mass and the clear sky index as follows:
+
+    .. math::
+
+        M = 1 + a_1(e^{-k_c}) + a_2(k_c-1)+a_3(AM-1.5),
+
+    where :math:`M` is the spectral mismatch factor, :math:`k_c` is the clear
+    sky index, :math:`AM` is the air mass, :math:`e` is Euler's number, and
+    :math:`a_1, a_2, a_3` are module-specific coefficients. In the JRC model
+    publication, the model used to estimate the air mass (denoted as :math:`AM`
+                                                          ) is not stated. The
+    clear sky index, which is the ratio of GHI to clear sky GHI, uses the ESRA
+    model [2]_ to estimate the clear sky GHI. Prior to this calculation, the
+    irradiance measurements are corrected for angle of incidence using the
+    Martin and Ruiz model [3]_.
+
+    References
+    ----------
+    .. [1] Huld, Thomas, Tony Sample, and Ewan D. Dunlop., 2009. A simple model
+        for estimating the influence of spectrum variations on PV performance.
+        In Proceedings of the 24th European Photovoltaic Solar Energy
+        Conference, Hamburg, Germany pp. 3385-3389. 2009.
+       :doi:`10.4229/24thEUPVSEC2009-4AV.3.27`
```

Comment:
Is it me or this DOI is broken?

Reply:
It's not you. The DOI points to a URL for the conference proceedings, which is extinct. EU PVSEC doesn't have the 2009 proceedings online, and I can only find them listed as hard copy in a [few libraries](https://search.worldcat.org/title/24th-European-Photovoltaic-Solar-Energy-Conference-and-Exhibition-:-Conference-21-25-September-2009-Exhibition-21-24-September-2009-2009-Hamburg-proceedings/oclc/503456868).

What to do? I would provide the citation as above, and instead of the broken DOI I would add "accessed at " and the URL for the copy on ResearchGate.

Reply:
I was supposed to leave a comment to ask about this as it was not working for me either so I also wanted to check. Thanks for picking up on it
Also thank you both for all the other review comments --- will revise accordingly